CITATION.cff

```diff
@@ -79,8 +79,8 @@ preferred-citation:
     - family-names: Su
       given-names: Yu
   title: "BioCLIP: A Vision Foundation Model for the Tree of Life"
-  year: 2023
-  journal: arXiv
+  year: 2024
+  collection-title: "Proceedings of IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR)"
   doi: "10.48550/arXiv.2311.18803"
```

Comment:
```suggestion
```
That's the arXiv DOI, so shouldn't be retained with this citation.

Also, line 55 should be `type: conference-paper`, not article. That _should_ make the citation generate properly on the sidebar.